### ISSUE TITLE
Default LV2 in stable image

### ISSUE THREAD

**Issue by riban-bw** [zynthian/zynthian-issue-tracking#418]
**Is your feature request related to a problem? Please describe.**
It would be advantageous to enable a set of LV2 plugins that showcase Zynthian in the default stable image.

**Describe the solution you'd like**
This table shows the list of available LV2 plugins. Let's identify those that are enabled by default.

Plugin | Enabled | Reason
------------ | ------------- | -------------
ADLplug||
AirFont320||
Black Pearl 4A||
Black Pearl 4B||
Black Pearl 5||
Black Pearl Drumkit||
Black Pearl Drumkit Multi||
Calf Fluidsynth||
Calf Monosynth|X|Great sounding mono synth, really good for, well, mono synth stuff, bass and leads
Calf Organ||
Calf Wavetable||
Dexed|X|Needs no introduction, great FM and tons of presets
DrMr Sampler||
DrumGizmo||
DrumSynth|X|Synthesized drums with a full octave of sounds, adsr and compressor included
Drumkits Percussion||
Example Metronome||
Exampler||
Fabla|X|Sampler/drum sampler - needs more presets
Fluid Bass||
Fluid Brass||
Fluid Chromatic Percussion||
Fluid Drums||
Fluid Ensemble||
Fluid Ethnic||
Fluid Guitars||
Fluid Organs||
Fluid Percussion||
Fluid Pianos||
Fluid Pipes||
Fluid Reeds||
Fluid SoundFX||
Fluid Strings||
Fluid SynthFX||
Fluid SynthLeads||
Fluid SynthPads||
FluidGM||
Foo YC20 Organ||
Geonkick||
Helm|X|Amazing plugin, can do almost anything, good for digital/harsh sounds
JuceOPL||
Kars||
LSP Multi-Sampler x12 DirectOut||
LSP Multi-Sampler x12 Stereo||
LSP Multi-Sampler x24 DirectOut||
LSP Multi-Sampler x24 Stereo||
LSP Multi-Sampler x48 DirectOut||
LSP Multi-Sampler x48 Stereo||
LSP Sampler Mono||
LSP Sampler Stereo||
MDA DX10||
MDA JX10|X|substractive soft-synth inspired by some Roland machines from the 80
MDA Piano||
MDA ePiano|X|Low resource nice sounding electric piano
MiniOPL3||
Nekobi||
Noize Mak3r|X|Classical subtractive synth, great for learning with some really good sounds if programmed well
OPNplug||
Obxd|X|eally really good emulation of Oberheim OB-X. Great for pads and strings/brass
Pianoteq 7 STAGE||
Raffo Synth|X|digital emulator of a minimoog synthesizer
Red Zeppelin 4||
Red Zeppelin 5||
Red Zeppelin Drumkit||
Red Zeppelin Drumkit Multi||
SO-404 Bass Synthesizer||
SO-666 Feedback Synthesizer||
SO-kl5 Piano Synthesizer||
Sorcer||
String machine|X|Lush string sounds
Surge||
Triceratops||
VL-Tone||
Vex|X|
Vitalium||
Wolpertinger||
ZynAddSubFX||
amsynth|X|Analog Modelling Synthesizer with a classic subtractive topology
drumkv1||
padthv1|X|
reMID||
samplv1||
setBfree DSP Tonewheel Organ||
synthv1|X|
the infamous cellular automaton synth||

Table needs to be updated and validated. Need to add tables for audio and MIDI effects.

**Comment by Pastitas** [comment]
I would add the midi clock generator for sure, it enables more complex midi setups and is basicly a standart offering with all midi devices

**Comment by riban-bw** [comment]
I agree we need MIDI clock output and certainly would have added to the list but the plug-in is flawed so we may add MIDI clock output native to Zynthian before the LV2 plugin is fixed.

**Comment by dcoredump** [comment]
I am using often **AirFont320** and some of th **Fluid-Synths**. They are nice for "background" sounds. Also Drum-Kits are useful. The Red* and Black* sound good, but for synth sounds I am using LS and the sampled drumkits. A kind of drum category would be useful...

**Comment by Pastitas** [comment]
I'd try to err on the side of a smaller default selection, given the fact that enabling them right away in webconf is an option
